doc/nrf/device_guides/working_with_nrf/nrf54l/ug_nrf54l15_gs.rst

```diff
@@ -56,299 +54,13 @@ Complete the following steps to test if the PDK works correctly:
-
-          1. Store your credentials (your username and the PAT created before) on disk using the ``store`` command from the git credential helper.
-
-             .. code-block:: shell
-
-                git config --global credential.helper store
-
-          #. Create a :file:`~/.git-credentials` (or :file:`%userprofile%\\.git-credentials` on Windows) and add this line to it::
-
-                https://<GitHub username>:<Personal Access Token>@github.com
-
-             See the `git-credential-store`_ manual page for details.
-
-          If you don't want to store any credentials on the file system, you can store them in memory temporarily using `git-credential-cache`_ instead.
-
-      .. tab:: SSH authentication
-
-          The `west manifest file`_ in the |NCS| uses ``https://`` URLs instead of ``ssh://``.
-          If you are already using `SSH-based authentication`_, you can reuse your SSH setup by adding the following to your :file:`~/.gitconfig` (or :file:`%userprofile%\\.gitconfig` on Windows):
-
-             .. parsed-literal::
-                :class: highlight
-
-                   [url "ssh://git@github.com"]
-                         insteadOf = https://github.com
-
-          This will rewrite the URLs on the fly so that Git uses ``ssh://`` for all network operations with GitHub.
-
-          You achieve the same result also using Git Credential Manager:
-
-          .. code-block:: shell
-
-                git config --global credential.helper store
-                git config --global url."git@github.com:".insteadOf "https://github.com/"
-
-          If your SSH key has no password, fetching should just work. If it does have a
-          password, you can avoid entering it manually every time using `ssh-agent`_.
-
-          On GitHub, see `Connecting to GitHub with SSH`_ for details on configuration
-          and key creation.
-
-#. Enter the following command to clone the project repositories::
-
-      west update
-
-   Depending on your connection, this might take some time.
-
-#. Export a :ref:`Zephyr CMake package <zephyr:cmake_pkg>`.
-   This allows CMake to automatically load the boilerplate code required for building |NCS| applications::
-
-      west zephyr-export
-
-Your directory structure now looks similar to this::
-
-    ncs-lcs/work-dir
-    |___ .west
-    |___ bootloader
-    |___ modules
-    |___ nrf
-    |___ nrfxlib
-    |___ zephyr
-    |___ ...
-
-This is a simplified structure preview.
-There are additional folders, and the structure might change over time.
+On your computer, install the `nRF Connect for Desktop`_.
+You must also install a terminal emulator, such as `nRF Connect Serial Terminal`_ (from the nRF Connect for Desktop application) or the nRF Terminal (part of the `nRF Connect for Visual Studio Code`_ extension).
 
 .. _ug_nrf54l15_gs_sample:
 
```

Comment:
Line 81/369: The standard command is ``west flash --erase``, as per https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/config_and_build/programming.html . Can we just link to the programming guide instead of writing the command here? Once a user does it once, it is not needed to mention this command here.